Index: linux-6.12.79/drivers/net/ethernet/airoha/airoha_eth.h
===================================================================
--- linux-6.12.79.orig/drivers/net/ethernet/airoha/airoha_eth.h
+++ linux-6.12.79/drivers/net/ethernet/airoha/airoha_eth.h
@@ -10,12 +10,14 @@
 #include <linux/debugfs.h>
 #include <linux/etherdevice.h>
 #include <linux/iopoll.h>
+#include <linux/bitfield.h>
 #include <linux/kernel.h>
 #include <linux/netdevice.h>
 #include <linux/reset.h>
 #include <linux/soc/airoha/airoha_offload.h>
 #include <net/dsa.h>
 
+#define __field_prep(mask, val) (((val) << (__bf_shf(mask))) & (mask))
 #define AIROHA_MAX_NUM_GDM_PORTS	4
 #define AIROHA_MAX_NUM_QDMA		2
 #define AIROHA_MAX_NUM_IRQ_BANKS	4
